package.json

```diff
@@ -34,31 +34,36 @@
     "url": "http://www.bentley.com"
   },
   "dependencies": {
+    "require-dir": "^1.2.0"
+  },
+  "peerDependencies": {
     "@typescript-eslint/eslint-plugin": "~5.55.0",
     "@typescript-eslint/parser": "~5.55.0",
-    "eslint-import-resolver-node": "0.3.4",
-    "eslint-import-resolver-typescript": "^2.7.1",
+    "eslint": "^8.36.0",
     "eslint-plugin-deprecation": "^1.3.3",
-    "eslint-plugin-import": "^2.23.4",
     "eslint-plugin-jam3": "^0.2.3",
     "eslint-plugin-jsdoc": "^35.1.3",
     "eslint-plugin-jsx-a11y": "^6.4.1",
-    "eslint-plugin-prefer-arrow": "^1.2.3",
     "eslint-plugin-react": "^7.32.2",
     "eslint-plugin-react-hooks": "^4.6.0",
-    "require-dir": "^1.2.0"
-  },
-  "peerDependencies": {
-    "eslint": "^8.36.0",
     "typescript": "^3.7.0 || ^4.0.0 || ^5.0.0"
   },
   "devDependencies": {
-    "@types/eslint": "~7.2.13",
+    "@typescript-eslint/eslint-plugin": "~5.55.0",
```

Comment:
Why are some packages declared in both dev and peer dependencies?

Reply:
shouldn't they be? Peer dependencies are not installed for the root package, only for packages you pull in as dependencies... unless there have been some more recent changes I'm not aware of

Reply:
Right. It does make sense, just threw me at first.

Reply:
I was recently educated that pnpm actually installs direct peerDependencies automatically :o

Reply:
ok I played around with it and noticed that it's a recent feature of pnpm, so you have to use pnpm v8 for it. That's out of scope for this PR.